opencda/scenario_testing/config_yaml/rl_dqn_town06_carla.yaml

```diff
@@ -184,9 +184,12 @@ scenario:
 
```

Comment:
question: if users want to change to other models dqn, do they need to change a lot of things?